forge-gui/res/editions/Alchemy Horizons Baldur's Gate.txt

```diff
@@ -337,3 +328,32 @@ A239 U A-Krydle of Baldur's Gate @Bryan Sola
 A243 M A-Minsc & Boo, Timeless Heroes @Andreas Zafiratos
 A259 C A-Lantern of Revealing @Eytan Zana
 A262 U A-Navigation Orb @Robin Olausson
+
+[conjured]
```

Comment:
Does this "add" new cards to the edition that will require images? We intentionally left out conjured cards for this reason

Reply:
Those cards where in the [cards] section before my update so I guess they were already requiring images didn't they?
I have put them under a new section to exclude them from Booster generation, that's all.

Reply:
Oh! they weren't in [cards] actually, I mixed up with the hag ones.

Reply:
The hags were in the edition yes but the rest of conjured were not. We typically don't include conjured because they "add" another version of the card that don't even have unique art and therefore cause bloat. Conjure by default will pull a version of that card from another set.

Reply:
In fact they seems to have unique art (for some of them at least).
https://www.cardmarket.com/en/Magic/Products/Singles/Commander-Legends-Battle-for-Baldurs-Gate/Lightning-Bolt

Reply:
Is the version of that card from another set is from a random set?

Reply:
That art is in CLB though.. https://scryfall.com/card/clb/187/lightning-bolt

I'd rather we stay away from adding conjured cards to editions

Reply:
Yes sure, I understand, will remove them, apart from the Hag ones

Reply:
Ok done, I have removed them.